renovate.json

```diff
@@ -9,7 +9,8 @@
     },
     {
       "matchPackagePatterns": ["^k8s.io", "^sigs.k8s.io"],
-      "groupName": "k8s modules"
+      "groupName": "k8s modules",
+      "allowedVersions": "<= 1.20.0"
```

Comment:
I think we want to use the latest patch version of 1.20. Will that work with this?

Also I think we need to add controller-runtime as we need to use 0.6.X WDYT?

See https://github.com/giantswarm/app-operator/pull/831

Reply:
Then I think this should be `< 1.21.0` to include all patches on 1.20. Thanks